azurecontainerapps/azurecontainerapps.ts

```diff
@@ -164,23 +258,34 @@ export class azurecontainerapps {
                 console.log(tl.loc('DefaultTargetPortMessage', targetPort));
             }
 
-            // Add the target port to the optional arguments array
-            if (!util.isNullOrEmpty(targetPort)) {
-                optionalCmdArgs.push(`--target-port ${targetPort}`);
+            // Set the target port to 80 if it was not provided or determined
+            // Note: this step should be skipped if we're using properties from the YAML configuration file (YAML defines target port with ingress)
+            if (ingressEnabled && util.isNullOrEmpty(targetPort) && !shouldUseYamlProperties) {
+                targetPort = '80';
```

Comment:
Could you please clarify if Container Apps itself is taking care of defining a default value for the target port? I'm trying to determine if there is a way for us to delegate that logic to them and not need to replicate some of the CLI logic (or logic that should be there if not already) on our side.

Reply:
They are not -- if you specify that you want ingress enabled for a Container App, you _must_ specify a target port. Previously with `az containerapp up`, ingress was enabled by default and if no target port was provided (or discovered), they would use `80`, so this is another example of reproducing what `up` is doing.